src/test/java/roomescape/MissionStepTest.java

```diff
@@ -1,20 +1,128 @@
 package roomescape;
 
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.DynamicTest.dynamicTest;
+
 import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DynamicTest;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.annotation.DirtiesContext;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 @DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
-public class MissionStepTest {
+class MissionStepTest {
 
     @Test
-    void 일단계() {
-        RestAssured.given().log().all()
-                .when().get("/")
-                .then().log().all()
+    @DisplayName("방탈출 어드민 메인 페이지 조회를 확인한다")
+    void showAdminMainPage() {
+        RestAssured.given()
+                .log()
+                .all()
+                .when()
+                .get("/admin")
+                .then()
+                .log()
+                .all()
                 .statusCode(200);
```

Comment:
```suggestion
        RestAssured.given().log().all()
                .when().get("/")
                .then().log().all()
                .statusCode(200);
```
지금 상황에서는 한 줄에 점 하나를 찍는 것이 오히려 가독성을 해치는 것 같아요.
이전과 같이 given, when, then 으로 어떤 상황인지 명확하게 구분되도록 하는건 어떨까요?

Reply:
이 방법이 더 가독성이 좋은 것 같네요! 감사합니다.